### ISSUE TITLE
Incorrect detection of summary

### ISSUE THREAD

**Issue by arogachev** [yiisoft/yii2-apidoc#283]
Extracted from #282. Reporter - @WinterSilence (https://github.com/yiisoft/yii2-apidoc/pull/282#issuecomment-1080587912).

**Comment by WinterSilence** [comment]
@arogachev I'm think [this regexp](https://regex101.com/r/qjSqHv/2) can match first sentence, but need more tests.

**Comment by arogachev** [comment]
Yes, we can try use regexp as well. Does it work with existing tests at least?

**Comment by WinterSilence** [comment]
@arogachev i'm create simple tests, left menu Functions > Unit tests

**Comment by WinterSilence** [comment]
@arogachev ups... previous links outdated, look test to [https://regex101.com/r/4MMD3n](https://regex101.com/r/4MMD3n/1)
